pkgs/top-level/all-packages.nix

```diff
@@ -9298,7 +9298,7 @@ with pkgs;
 
   grantlee = libsForQt5.callPackage ../development/libraries/grantlee { };
 
-  glib = callPackage ../development/libraries/glib (let
```

Comment:
You can remove this line

Reply:
So add a comment there explaining why it's necessary.

Reply:
There already is one.